language/de_x_sie/cli.php

```diff
@@ -88,11 +88,11 @@
 	'CLI_DESCRIPTION_THUMBNAIL_RECREATE'				=> 'Erstellt alle Vorschaubilder neu.',
 
 	'CLI_DESCRIPTION_UPDATE_CHECK'					=> 'Prüft, ob das Board auf dem neusten Stand ist.',
-	'CLI_DESCRIPTION_UPDATE_CHECK_ARGUMENT_1'		=> 'Name der zu überprüfenden Erweiterung (bei „all“, werden alle Erweiterungen überprüft)',
-	'CLI_DESCRIPTION_UPDATE_CHECK_OPTION_CACHE'		=> 'Führe den „check“-Befehl mit Cache aus.',
-	'CLI_DESCRIPTION_UPDATE_CHECK_OPTION_STABILITY'	=> 'Führe den „check“-Befehl mit der Auswahl „stable“- oder „unstable“-Versionen aus.',
+	'CLI_DESCRIPTION_UPDATE_CHECK_ARGUMENT_1'		=> 'Name der zu überprüfenden Erweiterung (bei „all“ werden alle Erweiterungen überprüft)',
+	'CLI_DESCRIPTION_UPDATE_CHECK_OPTION_CACHE'		=> 'Führt den „check“-Befehl auf Basis der im Cache gespeicherten Informationen aus.',
+	'CLI_DESCRIPTION_UPDATE_CHECK_OPTION_STABILITY'	=> 'Führt den „check“-Befehl mit der Auswahl „stable“- oder „unstable“-Stabilität aus.',
```

Comment:
"Version" in dieser Zeile war schon in Ordnung. Nur drunter nicht. Wenn du es gleich haben willst dann vielleicht besser
> Führt den "check"-Befehl wahlweise mit der Stabilität "stable" oder "unstable" aus.